test/functional/3rd-party/3rd-party-bundle-list-basic.bats

```diff
@@ -30,9 +30,9 @@ global_setup() {
 
 	assert_status_is "$SWUPD_OK"
 	expected_output=$(cat <<-EOM
-		_______________________
```

Comment:
Isn't it better to filter this from the output so we don't need to add that to tests?

Reply:
And in json_output the output is not very cool. We print one `_` per line

Reply:
> Isn't it better to filter this from the output so we don't need to add that to tests?

yeah, that may be a good idea. We can always use `--identical` for those tests we really want to check all output.

Reply:
> And in json_output the output is not very cool. We print one `_` per line

yes, it is not cool, but I didn't know how to fix this, currently there is no way of making exceptions of what we include in the json output. I am open to suggestions @otaviobp

Reply:
I made a PR trying to fix that: #1510

Reply:
+1 ☝️